src/table/utils/handle-scroll.ts

```diff
@@ -35,18 +35,18 @@ export const handleNavigateTop = (cellCoord, rootElement) => {
   if (x < MIN_ROW_COUNT) {
     tableContainer.scrollTo({
       top: 0,
-      behavior: 'instant',
+      behavior: 'auto',
     });
   } else {
-    const tableHead = rootElement.getElementsByClassName('sn-table-head-cell')[0];
+    const tableHead = rootElement.getElementsByClassName('sn-table-head-cell')[0] as HTMLElement;
     const rowElements = rootElement.getElementsByClassName('sn-table-row');
-    const cell = rowElements[x]?.getElementsByClassName('sn-table-cell')[y];
+    const cell = rowElements[x]?.getElementsByClassName('sn-table-cell')[y] as HTMLElement;
```

Comment:
the sanity check here is a bit strange. There is a `?.` and then assume that cell exist in the if block below. Is there a case where cell is undefined?

Reply:
rowElements = [rootElement,  rootElement];
rowElements[2] is undefined
undefiiend.?getElementsByClassName('sn-table-cell')[y] is undefined

Gonna fix